src/Mirza/Common/GS1BeamOrphans.hs

```diff
@@ -564,3 +570,38 @@ instance ToParamSchema EPC.LocationEPC where
   toParamSchema _ = mempty
     & type_ .~ SwaggerString
     & pattern ?~ "urn:epc:id:sgln:\\d+\\.\\d+(\\.\\d+)?"
+
+
+
+-- Should this be in GS1Combinators?
+newtype LabelEPCUrn = LabelEPCUrn {getLabelEPCUrn :: Text}
```

Comment:
The type was defined in `Mirza.SupplyChain.Types` before, but since the beam instances had to be defined, it was moved here (as `Mirza.SupplyChain.Types` imports this module).

I will move the definition into GS1Combinators in a different PR to keep this PR relevant :slightly_smiling_face: 
#359 has been opened regarding this